tests/testit/test-paths.R

```diff
@@ -72,3 +72,17 @@ assert("relative_path() works", {
   (relative_path(c('foo/bar.txt', 'foo/baz.txt'), 'foo/') %==% c("bar.txt", "baz.txt"))
   (relative_path('foo/bar.txt', 'foo') %==% "bar.txt")
 })
+
+assert("proj_root() works", {
+  dir.create(tmp_dir <- tempfile())
+  tmp_dir_slash <- paste0(tmp_dir, "/")
+  file.create(f1 <- file.path(tmp_dir, "test.Rproj"))
+  writeLines(c("Version: 1.2.3", "test: 321"), f1)
+  (proj_root(tmp_dir) %==% tmp_dir)
```

Comment:
The test is failing on Windows and macOS, e.g., https://github.com/yihui/xfun/pull/55/checks?check_run_id=2760307597